integration/src/main/resources/configs/integration/DeleteWorkspaceWithPolicy.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This test won't get run because it is not in any test suite. What that your intention?
It is tricky because we are not exposing our feature flags from WSM, so the test doesn't know if it should run or not.